### ISSUE TITLE
Source mapping problem with arrow functions

### ISSUE THREAD

**Issue by wehrstedt** [otris/vscode-janus-debug#180]
Script:
```js
var arr = [1, 2];
arr.forEach((elem) => {         // Step into here
	util.out(elem);
})
```

If you debug this script with `Upload and debug`, and you try to step into the arrow function (see comment above), the source mapping does not work.

A workaround:
You can set a breakpoint inside the arrow function and press `Continue`, then everything works fine

**Comment by tineheller** [comment]
First test shows, it has to be fixed here 11e802ce70afcf069741cce7c456594cc6604f80.
We must return a suitable frame to vscode but we have no file and line. Probably we should map the `'self-hosted'` frame to the current context frame and set a suitable line number.

**Comment by tineheller** [comment]
The project was moved to GitLab
https://gitlab.otris.de/tools/vscode-janus-debug/issues/180
